README.md

```diff
@@ -198,6 +198,36 @@ buildctl build ... --exporter=oci --exporter-opt output=path/to/output.tar
 buildctl build ... --exporter=oci > output.tar
 ```
 
+### Exporting/Importing build cache (not image itself)
+
+#### To/From registry
+
+```
+buildctl build ... --cache-exporter registry --cache-exporter-opt ref=localhost:5000/myrepo:buildcache
+buildctl build ... --cache-importer registry --cache-importer-opt ref=localhost:5000/myrepo:buildcache
+```
+
+#### To/From local filesystem
+
+```
+buildctl build ... --cache-exporter local --cache-exporter-opt output=path/to/output-dir
+buildctl build ... --cache-importer local --cache-importer-opt input=path/to/input-dir --cache-importer-opt digest=sha256:deadbeef
```

Comment:
How would you find this digest in regular workflow? Maybe we could add a "name" key that can be specified on both sides? Could probably keep `index.json` then and if name is specified then `foo.json` instead. Or create these as a symlinks to digest if you want to keep it cleaner.

Reply:
nit: `input` seems weird to me here. I get the `input/output` pairing but usually `src/dir/path` would be used in this context. `input` doesn't map to a directory in my brain for some reason.

Reply:
> Maybe we could add a "name" key that can be specified on both sides?

I'm planning to support generating `index.json` by default and to add the following opts:
* `index-path=path/to/dir/file.json` (default: `filepath.Join(outputDir, "index.json")`)
* `tag=latest` (default: `latest`)
* `no-index=true` (default: true)

Reply:
> nit: input seems weird to me here. 

How about `store`

Reply:
@AkihiroSuda what about the first question here?